parlai/core/torch_generator_agent.py

```diff
@@ -567,11 +581,13 @@ def _v2t(self, vec):
         new_vec = []
         if hasattr(vec, 'cpu'):
             vec = vec.cpu()
+        start = True
```

Comment:
I don't understand why we need `start`. Is that because the first index can be `END_IDX`? Could you explain in a comment?

Reply:
Think this was leftover by some testing, removed